automated/linux/perf/perf.sh

```diff
@@ -81,7 +83,11 @@ run_perf_stat() {
 run_perf_test() {
     # Test 'perf test'
     info_msg "Performing 'perf test'..."
-    perf test -v 2>&1 | tee "${RESULT_LOG}"
+    if [ "${RUN_TESTS_IN_PARALLEL}" = "False" ] || [ "${RUN_TESTS_IN_PARALLEL}" = "false" ]; then
```

Comment:
Can we just go with false, not "False". Maybe this check should just check for "true". 
```
if [ "${RUN_TESTS_IN_PARALLEL}" = "true" ]; then
<blah>
else
<blah>
fi
```

Reply:
Have updated for this.